.github/workflows/tests.yml

```diff
@@ -46,12 +46,12 @@ jobs:
 
       - uses: actions/setup-node@v2
         with:
-          node-version: 14
+          node-version: 16
           cache: 'npm'
           cache-dependency-path: '**/package-lock.json'
 
-      # Installs system dependencies needed for playwright.
-      - uses: microsoft/playwright-github-action@v1
+      - name: Install playwright deps
+        run: npx playwright install-deps
```

Comment:
Do you have a theory on why this helped? Leave a comment?

Reply:
I'm not convinced this helped. Instead I did this while I was updating the libraries since it was recommended: https://github.com/microsoft/playwright-github-action#%EF%B8%8F-you-dont-need-this-github-action-%EF%B8%8F

It's also symmetric with [lit.dev](https://github.com/lit/lit.dev/blob/main/.github/workflows/integration-tests.yml#L17).